.docs/content/1.concepts/0.index.md

```diff
@@ -0,0 +1,32 @@
+[![License: AGPL v3](https://img.shields.io/badge/License-AGPL_v3-green.svg)](https://www.gnu.org/licenses/agpl-3.0)
+# ArmoniK Core
```

Comment:
Add a empty line around each title (no matter the level)

Reply:
This improve readbility and because it's markdown, the compiler will be able to use it or not so this extra line will not change the final rendering.